helpers/truncate.js

```diff
@@ -25,7 +25,7 @@ const substring = require('stringz').substring;
  */
 const factory = globals => {
     return function(string, length) {
-        if (typeof string !== 'string') {
+        if (typeof string !== 'string' || string.length === 0) {
```

Comment:
So if the `string` is an array or object (is it possible case?) we just return it. What if it such an array/object is too big?

Reply:
that's right, that's part of user input validation. The helper supports only non-empty strings. 
What do you mean by too big array/object? The data is either returned by backend or determined by user (meaning in templates). We have limitation on bundle size ( and I think on a template size)